demonstrator/ogc-api-processes/main.js

```diff
@@ -7,8 +7,64 @@ let processId = '';
 // The OpenLayers Draw interaction
 let olDraw;
 
-// The EPSG code of Webmercator
-const epsg3857 = 'EPSG:3857';
+// apply the projection definition for EPSG:3035
+proj4.defs('EPSG:3035','+proj=laea +lat_0=52 +lon_0=10 +x_0=4321000 +y_0=3210000 ' +
+  '+ellps=GRS80 +towgs84=0,0,0,0,0,0,0 +units=m +no_defs +type=crs');
+ol.proj.proj4.register(proj4);
+
+// setting up the channel sliders
+let minValue = 0;
+let maxValue = 56;
+let channel1Active = true;
+let channel2Active = false;
+let channel3Active = false;
+const origin = window.location.origin.indexOf('localhost') > -1 ?
```

Comment:
maybe easier to read like this: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes

```suggestion
const origin = window.location.origin.includes( 'localhost') ?
```